sdk/typescript/src/keypairs/passkey/index.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+export { PasskeyKeypair } from './keypair.js';
+export { PasskeyPublicKey } from './publickey.js';
```

Comment:
We should export the browser provider, the browser provider options, and the provider interface